src/app/Instance/CreateInstance/CreateInstance.tsx

```diff
@@ -69,11 +69,13 @@ const CreateInstance = (props: CreateInstanceProps): JSX.Element => {
     <Modal
       isOpen={isModalOpen}
       title={t("instance.createASEInstance")}
+      ouiaId="create-instance-modal"
```

Comment:
```suggestion
      ouiaId="create-instance"
```
Information that this is a modal is present in component type.